src/components/backgroundselector/BackgroundService.js

```diff
@@ -17,12 +17,13 @@ goog.require('ga_permalink');
     this.$get = function($rootScope, $q, gaTopic, gaLayers, gaPermalink) {
       var isOfflineToOnline = false;
       var bg; // The current background
-      var bgs = [ // The list of backgrounds available
-        {id: 'ch.swisstopo.swissimage', label: 'bg_luftbild', disable3d: true},
-        {id: 'ch.swisstopo.pixelkarte-farbe', label: 'bg_pixel_color'},
-        {id: 'ch.swisstopo.pixelkarte-grau', label: 'bg_pixel_grey'},
-        {id: 'voidLayer', label: 'void_layer'}
-      ];
+      var bgs = []; // The list of backgrounds available
+      var voidLayer = {id: 'voidLayer', label: 'void_layer'};
+      var bodIdToFrontLabel = {
+        'ch.swisstopo.swissimage': 'bg_luftbild',
+        'ch.swisstopo.pixelkarte-farbe': 'bg_pixel_color',
+        'ch.swisstopo.pixelkarte-grau': 'bg_pixel_grey'
+      };
```

Comment:
I think you should keep the array but rename it to `predefinedBgs`  then use it l58 instead of getting only the label .

With this array we could modifiy more easily the hardcoded values like `label` or `disable3d` in the future.